cfme/fixtures/pytest_selenium.py

```diff
@@ -929,6 +929,41 @@ def __getitem__(self, item):
                     repr(item), repr(self.keys())))
 
 
+def ssui_force_navigate(page_name, _tries=0, *args, **kwargs):
+    if _tries > 2:
+        # Need at least three tries:
+        # 1: login_admin handles an alert or CannotContinueWithNavigation appears.
+        # 2: Everything should work. If not, NavigationError.
+        raise exceptions.NavigationError(page_name)
+
+    if "context" in kwargs:
+        if not isinstance(kwargs["context"], ContextWrapper) and isinstance(
+                kwargs["context"], dict):
+            kwargs["context"] = ContextWrapper(kwargs["context"])
+
+    _tries += 1
```

Comment:
i just noticed that tries doesn't seem to be used internally
i wonder if it should be factored out (it seems tobe the same in normal force navigate

@psav @mfalesni